UI/aoc-app/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "aoc-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-scripts": "3.4.0"
```

Comment:
Why did you have to change this from its initial value of `4.0.4`?

Reply:
The newest react version wasn't compatible with the Heroku buildpack.